se/data/templates/onix-white-label.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think if we're going to provide a white label version of this file, we should basically leave it empty, because if the publisher doesn't look at it, they may inadvertently create an ebook that is not accessible, but with an onix file saying that it is.

Reply:
Sure, that’s reasonable.